keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -339,32 +632,43 @@ The below command should show maxUnavailable value:
 kubectl get statefulsets -o yaml | grep maxUnavailable
 ```
 
+Or refer to the new metric `rolling-update-duration`, it should exist.
+
 ###### How can someone using this feature know that it is working for their instance?
-TODO when we reach Beta
+
+With feature enabled, set the `maxUnavailable` great than 1, and pay attention to the rolling update pods at a time,
+it should equal to the `maxUnavailable`.
+Or when setting the `maxUnavailable` great than 1, the `rolling-update-duration` should decrease.
 
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
+I think it has little relevance with SLOs, but rolling update at a very low speed which impacts the running services.
```

Comment:
What kind of values of `rolling-update-duration` metric you expect to be normal, for example?

Reply:
A statefulset with 2 replicas is incomparable with a statefulset with 20 replicas, it makes more sense as self-comparison, like `maxUnavailable=4` should roll update more quickly then `maxUnavailable=2`.

Reply:
Actually, I don't know how long is slow, we don't have any measurement about this, and the update duration impacts by a lot of factors, e.g. the node pressure, even the container hooks.

Reply:
You don't have to be perfect with the proposed SLO, but I think we need to expose that information. We can reach out to sig-instrumentation for advice when implementing the metric itself and update this enhancement afterward. But don't say it has little relevance :wink:

Reply:
Updated.